eth2/types/src/u64_macros.rs

```diff
@@ -0,0 +1,610 @@
+// Copied from the slot_epoch_macros.rs file in https://github.com/sigp/lighthouse with some small
+// modifications.  Specifically, updated ssz macro to match the eth2_ssz crate's Encode trait, which
+// is slightly different than the Encode trait used internally in the sigp/lighthouse repo.
+
```

Comment:
Did you make any changes to this? You should ask lighthouse to publish this in particular and just import it? Any rationale behind not doing that for pieces you do not alter?

Reply:
Yes, I did.  Details are in the comment.  I'm open to using a crate here but not sure how generally useful it will be so not sure it makes sense to push as a crate.  Either way works for me though